solutions/telemetry/quarkus/producer/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '2.1'
```

Comment:
Do we need this docker-compose?

Reply:
Oh, I forgot about this initial test. Removed